Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM python:3
+
+# Install numpy using system package manager
+RUN apt-get -y update && apt-get -y install libav-tools imagemagick libopencv-dev python-opencv
+
+# Install some special fonts we use in testing, etc..
+RUN apt-get -y install fonts-liberation
+
+RUN apt-get install -y locales && \
+    locale-gen C.UTF-8 && \
+    /usr/sbin/update-locale LANG=C.UTF-8
+
+ENV LC_ALL C.UTF-8
+
+# do we need all of these, maybe remove some of them?
+RUN pip install imageio numpy scipy matplotlib pandas sympy nose decorator tqdm pillow pytest
+
+# install scikit-image after the other deps, it doesn't cause errors this way.
+RUN pip install scikit-image sklearn
+
+RUN git clone https://github.com/Zulko/moviepy.git /var/src/moviepy
+RUN cd /var/src/moviepy/ && python setup.py install
```

Comment:
Re-commenting since it is no longer showing up.

Since the Dockerfile is going to be included with the source, we won't need to pull down from git, we can just include it like this.

```
ADD . /var/src/moviepy/
RUN cd /var/src/moviepy/ && python setup.py install
```